homeassistant/components/mqtt_room/sensor.py

```diff
@@ -133,9 +133,7 @@ def message_received(msg):
                     ):
                         update_state(**device)
 
-        return await mqtt.async_subscribe(
```

Comment:
` async_added_to_hass` does not accept a return value. Reloading is not supported for this integration.
I had to remove the `return` to avoid mypy errors.